src/player/C4Player.cpp

```diff
@@ -726,29 +743,6 @@ void C4Player::SetFoW(bool fEnable)
 	fFogOfWar = fEnable;
 }
 
-bool C4Player::DoWealth(int32_t iChange)
-{
-	if (LocalControl)
-	{
-		if (iChange>0) StartSoundEffect("UI::Cash");
-		if (iChange<0) StartSoundEffect("UI::UnCash");
-	}
-	SetWealth(Wealth+iChange);
-
-	return true;
-}
-
-bool C4Player::SetWealth(int32_t iVal)
-{
-	if (iVal == Wealth) return true;
-
-	Wealth=Clamp<int32_t>(iVal,0,1000000000);
-
-	::Game.GRBroadcast(PSF_OnWealthChanged,&C4AulParSet(Number));
```

Comment:
The diff says that the line was removed, or did you look at the script file where wealth is managed? The HUD does not update without the callback.

Reply:
I think you didn't remove the definition of that constant from the header file (C4GameScript.h), but maybe I just missed it?

Reply:
Removed it now, but I left it in the script documentation because the script function in Player.c still does the callback. Otherwise the HUD would not update correctly.